app/services/embedded_content_finder_service.rb

```diff
@@ -9,13 +9,16 @@ def fetch_linked_content_ids(details)
 
   def find_content_references(value)
     case value
-    when Array
+    in [*, { content_type: "text/govspeak", content: String => content }, *]
```

Comment:
question: Why special-case text/govspeak rather than text/html? Are we sure that one or the other will always exist, so that it's safe not to check the alternative?